python/smqtk/utils/image_utils.py

```diff
@@ -0,0 +1,54 @@
+import io
+import PIL.Image
+
+from smqtk.representation.data_element.file_element import DataFileElement
+
+
+def is_loadable_image(dfe):
+    """
+    Determine if an image is able to be loaded by PIL.
+
+    :param dfe: A data file element to check
+    :type dfe: DataFileElement
+
+    :return: Whether or not the image is loadable
+    :rtype bool:
+
+    """
+    try:
+        PIL.Image.open(io.BytesIO(dfe.get_bytes()))
+        return True
+    except IOError, ex:
+        return False
```

Comment:
Define a ``log = logging.getLogger(__name__)`` and reinstate the removed warning here.